### PR TITLE
Dex's name card

### PR DESCRIPTION
**Pull request by Dex-theDev**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -115,6 +115,37 @@ <h2>A project for first-time contributions</h2>
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
 
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
```
Comment (dilshankarunarathne) [inline index.html:118]: Please change this 
`<!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________ -->` comment to
`<!-- ________ YourName Contributor card START ________ -->`

File: index.html [hunk 2]
```diff
@@ -115,6 +115,37 @@ <h2>A project for first-time contributions</h2>
...
+              </li>
+              <li>
+                <a href="https://roadmap.sh/" target="_blank" title="Raodmap">
+                  Roadmap - A guide for web development careers
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <!-- ________*END TEMPLATE* Contributor card END ________  -->
```
Comment (dilshankarunarathne) [inline index.html:147]: Please change this 
`<!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________ -->` comment to
`<!-- ________ YourName Contributor card START ________ -->` as well.

File: index.html (hunk 1 above)
Comment (dilshankarunarathne) [reply index.html:118]: This comment helps others to identify your card in the HTML file.

**Review by dilshankarunarathne — CHANGES_REQUESTED** [review]
Hello @Dex-theDev, please make these changes and make a pull request again.
We will review and merge your contribution ASAP.

**Comment by Dex-theDev** [comment]
i added my name as requested and pushed again.

**Review by dilshankarunarathne — APPROVED** [review]
Your code looks good. It will be merged ASAP.

**Comment by dilshankarunarathne** [comment]
Congratulations 🎉❤
Your pull request was merged. 
You will be able to see your card on the [project website](https://syknapse.github.io/Contribute-To-This-Project/).

**Pull request merged**